<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWorkSpace</class>
 <widget class="QDialog" name="CWorkSpace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabMain">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabNumbersRegistry">
      <attribute name="title">
       <string>&amp;Реестр номеров ЛН</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="0" colspan="2">
        <widget class="CTableView" name="tblNumbersRegistry"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="btnRequestNumbers">
         <property name="text">
          <string>&amp;Запросить номера в сервисе СФР</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lblSizeOfNumbersRegistry">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabReadyToSend">
      <attribute name="title">
       <string>&amp;Готово к передаче</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0" rowspan="3">
        <widget class="QGroupBox" name="grpReadyFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Фильтр</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="5" column="0" colspan="2">
           <widget class="QLineEdit" name="edtReadyNumber"/>
          </item>
          <item row="8" column="0" colspan="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="CApplyResetDialogButtonBox" name="buttonBoxReady">
            <property name="standardButtons">
             <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="COrgStructureComboBox" name="cmbReadyOrgStructure"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblReadyBegDate">
            <property name="text">
             <string>&amp;Дата c</string>
            </property>
            <property name="buddy">
             <cstring>edtReadyBegDate</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="lblReadyPerson">
            <property name="text">
             <string>&amp;Врач</string>
            </property>
            <property name="buddy">
             <cstring>cmbReadyPerson</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="lblReadyOrgStructure">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Под&amp;разделение</string>
            </property>
            <property name="buddy">
             <cstring>cmbReadyOrgStructure</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="CDateEdit" name="edtReadyBegDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lblReadyEndDate">
            <property name="text">
             <string>&amp;по</string>
            </property>
            <property name="buddy">
             <cstring>edtReadyEndDate</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="CDateEdit" name="edtReadyEndDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="lblReadyNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Номер</string>
            </property>
            <property name="buddy">
             <cstring>edtReadyNumber</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="CPersonComboBoxEx" name="cmbReadyPerson"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QPushButton" name="btnSend">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&amp;Передать в СФР</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="CTableView" name="tblReadyTempInvalidDocuments"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLabel" name="lblRowsInReadyTempInvalidDocuments">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExpired">
      <attribute name="title">
       <string>&amp;Контроль открытых ЭЛН</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="1">
        <widget class="CTableView" name="tblExpiredTempInvalidDocuments"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btnCheckExpiredDocuments">
         <property name="text">
          <string>&amp;Запросить состояние в СФР</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QGroupBox" name="grpExpiredFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Фильтр</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="8" column="0" colspan="2">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="lblExpiredOrgStructure">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Под&amp;разделение</string>
            </property>
            <property name="buddy">
             <cstring>cmbReadyOrgStructure</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="COrgStructureComboBox" name="cmbExpiredOrgStructure">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="lblExpiredPerson">
            <property name="text">
             <string>&amp;Врач</string>
            </property>
            <property name="buddy">
             <cstring>cmbHistoryPerson</cstring>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="CApplyResetDialogButtonBox" name="buttonBoxExpired">
            <property name="standardButtons">
             <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="lblExpiredDays">
            <property name="text">
             <string>&amp;Просрочено дней, более </string>
            </property>
            <property name="buddy">
             <cstring>edtExpiredDays</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QSpinBox" name="edtExpiredDays">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="CPersonComboBoxEx" name="cmbExpiredPerson"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblExpiredFullName">
            <property name="text">
             <string>&amp;ФИО получателя</string>
            </property>
            <property name="buddy">
             <cstring>edtExpiredFullName</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QLineEdit" name="edtExpiredFullName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblRowsInExpiredTempInvalidDocuments">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>…</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabHistory">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>&amp;История обмена</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0" rowspan="3">
        <widget class="QGroupBox" name="grpHistoryFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Фильтр</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="6" column="0">
           <widget class="QLabel" name="lblHistoryBegDate">
            <property name="text">
             <string>&amp;Дата c</string>
            </property>
            <property name="buddy">
             <cstring>edtReadyBegDate</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="CDateEdit" name="edtHistoryBegDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lblHistoryEndDate">
            <property name="text">
             <string>&amp;по</string>
            </property>
            <property name="buddy">
             <cstring>edtReadyEndDate</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="1">
           <widget class="CDateEdit" name="edtHistoryEndDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="lblHistoryNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Номер</string>
            </property>
            <property name="buddy">
             <cstring>edtReadyNumber</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLineEdit" name="edtHistoryNumber"/>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="CApplyResetDialogButtonBox" name="buttonBoxHistory">
            <property name="standardButtons">
             <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="COrgStructureComboBox" name="cmbHistoryOrgStructure">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="lblHistoryOrgStructure">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Под&amp;разделение</string>
            </property>
            <property name="buddy">
             <cstring>cmbReadyOrgStructure</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="lblHistoryPerson">
            <property name="text">
             <string>&amp;Врач</string>
            </property>
            <property name="buddy">
             <cstring>cmbHistoryPerson</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="CPersonComboBoxEx" name="cmbHistoryPerson"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="CInDocTableView" name="tblTempInvalidDocumentExports"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CApplyResetDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>library/DialogButtonBox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabMain</tabstop>
  <tabstop>tblNumbersRegistry</tabstop>
  <tabstop>btnRequestNumbers</tabstop>
  <tabstop>cmbReadyOrgStructure</tabstop>
  <tabstop>cmbReadyPerson</tabstop>
  <tabstop>edtReadyNumber</tabstop>
  <tabstop>edtReadyBegDate</tabstop>
  <tabstop>edtReadyEndDate</tabstop>
  <tabstop>buttonBoxReady</tabstop>
  <tabstop>tblReadyTempInvalidDocuments</tabstop>
  <tabstop>btnSend</tabstop>
  <tabstop>cmbHistoryOrgStructure</tabstop>
  <tabstop>cmbHistoryPerson</tabstop>
  <tabstop>edtHistoryNumber</tabstop>
  <tabstop>edtHistoryBegDate</tabstop>
  <tabstop>edtHistoryEndDate</tabstop>
  <tabstop>buttonBoxHistory</tabstop>
  <tabstop>tblTempInvalidDocumentExports</tabstop>
  <tabstop>cmbExpiredOrgStructure</tabstop>
  <tabstop>cmbExpiredPerson</tabstop>
  <tabstop>edtExpiredDays</tabstop>
  <tabstop>buttonBoxExpired</tabstop>
  <tabstop>tblExpiredTempInvalidDocuments</tabstop>
  <tabstop>btnCheckExpiredDocuments</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
